service/filter.go

```diff
@@ -68,9 +70,9 @@ func ParseRequestForm() Filter {
 		ct, err := ContentType(req)
```

Comment:
What is `pseudo filter`?  ParseRequestForm will return both true and false in different conditions, is it still a pseudo filter?
> ParseRequestForm is a pseudo filter.

Reply:
Fixed comment.